app/assets/stylesheets/news.scss

```diff
@@ -0,0 +1,63 @@
+.index_news_btns {
```

Comment:
Please discuss styles with @anastasiabalaban, you both have some general styles, that are duplicated.
All your `scss` files will be compiled into one `application.css` and only the latest rool in that file will be applied. So duplicated general styles should be moved to some `general.scss` file